src/Tests/TestExtensions.cs

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.IQSharp;
+using Microsoft.Quantum.IQSharp.AzureClient;
+using Microsoft.Quantum.IQSharp.Kernel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Tests.IQSharp
+{
+    
+    internal static class TestExtensions
+    {
+        internal class InputAssert : UsingEngineAssert
+        {
+            internal string Code;
+            internal int? CursorPos;
+        }
+
+        internal class UsingEngineAssert
+        {
+            internal IQSharpEngine Engine;
+        }
+        internal static async Task<T> WithMockAzure<T>(this Task<T> engine)
```

Comment:
nitpick: line space.

Reply:
Ah, good catch, thank you.
```suggestion

        internal static async Task<T> WithMockAzure<T>(this Task<T> engine)
```